Week2/homework/maartjes-work.js

```diff
@@ -47,15 +47,23 @@ const maartjesHourlyRate = 20;
 
 function computeEarnings(tasks, hourlyRate) {
   // Replace this comment and the next line with your code
+  //Map the tasks to durations in hours.
+  const durations= maartjesTasks.map(tasks=> tasks.duration);
+  const durationsInHours = durations.map(durations =>durations/60);
+  //Filter out everything that took less than two hours (i.e., remove from the collection)
+  const plus2Hours = durationsInHours.filter( val => val >= 2);
+  //Multiply the each duration by a per-hour rate for billing (use €20/hour) and sum it all up.
+  const totalSum =plus2Hours.reduce(function (accumulator, currentValue) {
+     return accumulator + (currentValue * maartjesHourlyRate);;
```

Comment:
Try rewriting this using an arrow function. It is more readable in the case of short functions like this one.
Also you made something dangerous here: is `maartjesHourlyRate` the best variable to use here? What would happen if you moved the declaration of the function at the top of the file? Would it still work?